.github/workflows/main.yaml

```diff
@@ -10,11 +10,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - id: read-version
-        uses: NiklasLehnfeld/flutter-version-number-action@main
+        uses: blackfoot-makers/flutter-version-number-action@main
```

Comment:
With This change the Workflow Which Should test This action is Not using This Action anymore - Revert it

Reply:
Hey yes, sorry, i wanted to use it in my project so i had to change it in my repository, i forgot about this pull request sorry, i'll close it.